src/openrct2/ride/Vehicle.cpp

```diff
@@ -5207,7 +5207,11 @@ static void vehicle_kill_all_passengers(Vehicle* vehicle)
     if (numFatalities != 0)
     {
         ride->FormatNameTo(gCommonFormatArgs + 2);
```

Comment:
This can probably be moved inside the `if` as well.

Reply:
Good point.